project-base/tests/FrontendApiBundle/Functional/Product/AllProductsTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\FrontendApiBundle\Functional\Product;
+
+use Tests\FrontendApiBundle\Test\GraphQlTestCase;
+
+class AllProductsTest extends GraphQlTestCase
+{
+    public function testFirstFiveProductsWithName(): void
+    {
+        $query = '
+            query {
+                products(limit: 5) {
+                    name
+                }
+            }
+        ';
+
+        $arrayExpected = [
+            'data' => [
+                'products' => [
+                    ['name' => '22" Sencor SLE 22F46DM4 HELLO KITTY'],
```

Comment:
This should be translated

Reply:
Ok, I fixed it https://github.com/shopsys/shopsys/pull/1471/commits/65bdd61dc580d825ae04a9cf8785fb2093f17232